### PR TITLE
[WIP] Open Science Fair 2019 demo & poster proposals

### LINKED ISSUE #434: Turing Way proposal for the Open Science Fair
### Summary

Apply to present a poster or demo under Training and skills for Open Science at the [Open Science Fair](https://www.opensciencefair.eu/), September 16-18, 2019 in Porto, Portugal.

The deadline for submissions is May 17, 2019.

### What needs to be done?

Authors are asked to submit a short proposal (300 - 500 words) that describes the main contributions of the Poster / Demo. Proposals should contain a brief abstract, place an emphasis on the motivation for the work, and summarize contributions being presented. For Posters, preliminary results may also be included.

Proposal templates and criteria here: https://www.opensciencefair.eu/call-for-posters-demos-2019

### Who can help?

---

### PR DESCRIPTION
**Pull request by rainsworth**
### Summary

Related to #434  

This pull request includes the proposal submissions to present a poster and/or demo under Training and skills for Open Science at the [Open Science Fair](https://www.opensciencefair.eu/), September 16-18, 2019 in Porto, Portugal.

The deadline for submissions has been extended to 22 May 2019!

### List of changes proposed in this PR (pull-request)

- [x] Complete poster submission template
- [x] Complete demo submission template

### What should a reviewer concentrate their feedback on?

- [x] Anything else to add?
- [x] Grammar and typos.

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).

### CODE REVIEW

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# OSFair2019 Demo Submission 
+Open Science Fair 2019, September 16-18, Porto, Portugal
+
+*For Authors: remove text in italics and replace with your own content.*
+
+## The Turing Way: A handbook for reproducible data science 
+
+*Author 1 name, affiliation, email address; Author 2 name, affiliation, email address*
+
+The Turing Way, Alan Turing Institute; Rachael Ainsworth, University of Manchester, rachael.ainsworth@manchester.ac.uk
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: I don't mind you being the author for a demo presentation @rainsworth. Up to you if you want to leave the Turing Way in, but - for example - when I give talks I just have myself as the "author".

See, for example: https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/ukoswg_submission.md#authors

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# OSFair2019 Demo Submission 
+Open Science Fair 2019, September 16-18, Porto, Portugal
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: Can you add details in the bullet points as in https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/ukoswg_submission.md#uk-oswg-uk-network-of-open-science-working-groups ?

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+*Author 1 name, affiliation, email address; Author 2 name, affiliation, email address*
+
+The Turing Way, Alan Turing Institute; Rachael Ainsworth, University of Manchester, rachael.ainsworth@manchester.ac.uk
+
+## Abstract
+
+*Summary of your proposal; maximum 300 words. The abstract should be a concise summary of the demo content and the relation to the overall objective of the conference. Please copy and paste this into the submission system at the time of submission.*
+
+Reproducible research is necessary to ensure that scientific work can be trusted. Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists.
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: Please can you make all sentences separate lines?

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+Reproducible research is necessary to ensure that scientific work can be trusted. Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. 
+
+*The Turing Way* is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". It includes training material on version control, analysis testing, and open and transparent communication with future users, and builds on Alan Turing Institute case studies and workshops. This project is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way.
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: Add in link to netlify now that we have it?

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+*The Turing Way* is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". It includes training material on version control, analysis testing, and open and transparent communication with future users, and builds on Alan Turing Institute case studies and workshops. This project is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way. 
+
+During this session, we will lead a collaborative review of the content so far and show Open Science Fair participants how they can contribute their knowledge to make it even better going forwards. This demo relates to the overall theme of the conference, as the *Turing Way* provides the tools to improve research habits in a self-contained handbook. It will also ensure that PhD students, postdocs, PIs and funding teams know which parts of the "responsibility of reproducibility" they can affect, and what they should do to nudge data science to being more efficient, effective and understandable.
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: I think this sounds great - thank you for submitting the application!!

I don't know if the first two paragraphs really emphasise the community angle as much as they could for this audience. I feel like the sentence about it being openly developed needs to be brought front and centre for this event. What do you think?

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+*	FAIR data policy and practice: from theory to implementation
+*	EOSC in national settings
+*	Thematic Clouds 
+
+RRI and Open Science: bridging the gap
+*	Citizen Science and Public and Societal Engagement
+*	 Governance settings for Institutional embedded RRI and OS
+
+__Training and skills for Open Science__
+*	__Sustaining Open Science training: people and resources__
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: ✅ Coolio 👾

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+*List 3-4 key terms that describe the subject of the proposal.*
+
+Reproducibile research, data science, open science, open research.
+
+## Audience
+
+*Identify the target audience. Some examples might be: Policy makers and funders, researchers, research Infrastructures and research communities, repository managers, publishers and content providers, libraries, research administrators, service providers and innovators.*
+
+Policy makers and funders, researchers, research Infrastructures and research communities, repository managers, publishers and content providers, libraries, research administrators.
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: Weird capitalisation on "Infrastructures"?

Maybe emphasise the different types of researchers (ECRs, research software engineers and senior investigators?)

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+
+## Demo presentation content
+
+*Authors are asked to submit a short proposal that describes the main contributions of the demo. Proposals should contain a brief abstract, place an emphasis on the motivation for the work, and summarize contributions being presented.*
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: I'm so confused about the difference between this and the abstract above? 😕

File: conferences/abstracts/OSF19_Demo_submission.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+The project also demonstrates open and transparent project management and communication with future users, as it is openly developed at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way. Any and all questions, comments and recommendations are welcome. All resources for the "Boost your research reproducibility with Binder" and "Build your own BinderHub" workshops we have delivered, as well as how to organise a Book Dash (a one day book sprint), are also open and will soon be submitted to the Journal of Open Source Education (JOSE).
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Demo_submission.md]: This is a great paragraph. And basically the thing I was saying to emphasise in my comment above 😆

File: conferences/abstracts/OSF19_Poster_submission.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
+# OSFair2019 Poster Submission  
+Open Science Fair 2019, September 16-18, Porto, Portugal
+
+*For Authors: remove text in italics and replace with your own content.*
+
+## The Turing Way: A handbook for reproducible data science 
+
+*Author 1 name, affiliation, email address; Author 2 name, affiliation, email address*
+
+The Turing Way, Alan Turing Institute; Rachael Ainsworth, University of Manchester, rachael.ainsworth@manchester.ac.uk
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Poster_submission.md]: For posters, please can we have all the authors. Fine for now to list the authors who were on the poster I made (https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/presentations/TPS_ScopingWorkshop_20190321/Whitaker_TuringWayPoster_TPSScopingWorkshop.pdf) although if you see key people who should also be authors please add them.

File: conferences/abstracts/OSF19_Poster_submission.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+*List 3-4 key terms that describe the subject of the proposal.*
+
+Reproducibile research, data science, open science, open research.
+
+## Audience
+
+*Identify the target audience. Some examples might be: Policy makers and funders, researchers, research Infrastructures and research communities, repository managers, publishers and content providers, libraries, research administrators, service providers and innovators.*
+
+Policy makers and funders, researchers, research Infrastructures and research communities, repository managers, publishers and content providers, libraries, research administrators.
```
Comment (KirstieJane) [inline conferences/abstracts/OSF19_Poster_submission.md]: Same comment as above 😄

File: conferences/abstracts/OSF19_Demo_submission.md (hunk 8 above)
Comment (rainsworth) [reply conferences/abstracts/OSF19_Demo_submission.md]: Same! 🙈  The abstract submission portal asks for the abstract and to attach a pdf "paper" which I think is the word doc they have on the website https://www.opensciencefair.eu/call-for-posters-demos-2019

Just trying to cover all bases, but will definitely re-emphasise some of the extra points made in the demo content up in the abstract like you suggest!

**Pull request merged**